modules/openapi-generator/src/main/resources/swift5/APIs.mustache

```diff
@@ -5,16 +5,25 @@
 //
 
 import Foundation
+{{#useVapor}}
+import Vapor
+{{/useVapor}}
 
-{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}open{{/nonPublicApi}} class {{projectName}}API {
```

Comment:
Hi, could you explain the why of this change?

Reply:
I figured it's extraneous to have `API` added to the project name since in most cases the project name _is_ in fact the library/class name.

Reply:
This is a breaking change the affects a lot of files, so I suggest to revert it.